the-collab-lab/tcl-38-smart-shopping-list
# km - updated style home page and add item page with desktop breakpoints

## Description

This PR updates the breakpoints for the Home page and the Add Item page to be responsive for viewing the app on larger screens.

## Related Issue

Closes #60. 

## Acceptance Criteria

Refine the responsiveness for mobile and tablet/desktop breakpoints. The layout should match the Figma design regardless of device screen size. Note: This issue will create desktop breakpoints only, but desktop-specific styling will be addressed in a separate issue.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

| Home | Add Item |
| ---- | -------|
| ![image](https://user-images.githubusercontent.com/40015502/156904509-329c08dd-a9fc-4296-a87a-cf46c935b37d.png) | ![image](https://user-images.githubusercontent.com/40015502/156904607-02f67637-1e5f-45c1-9c6c-033756efa7ce.png) |


### After

| Home | Add Item |
| ---- | -------|
| ![image](https://user-images.githubusercontent.com/40015502/156904520-d537af0f-2e1b-4659-9a5e-b5a0acec99fa.png)| ![image](https://user-images.githubusercontent.com/40015502/156904564-78228163-9a8d-404e-bfad-02577404fd9b.png) |


## Testing Steps / QA Criteria

Pull down branch and check to see if styling matches the above, does not alter mobile views, and that no functionality has been disrupted. Please note, the About/Share Token buttons have been styled in another branch and will be merged with this branch.
